lib/versioning/composer/index.spec.ts

```diff
@@ -12,6 +12,11 @@ describe('semver.equals(a, b)', () => {
   it('should pad really short version', () => {
     expect(semver.equals('v1.0.0', '1')).toBe(true);
   });
+  it('should translate stability modifier', () => {
+    expect(semver.equals('1.0@alpha3', '1.0.0-alpha.3')).toBe(true);
```

Comment:
@hussainweb It shouldn't include the patch
```suggestion
    expect(semver.equals('1.0@alpha3', '1.0-alpha.3')).toBe(true);
```

Reply:
So, there is a check before which pads version string with zeroes. Although, now I see that that replacement may not be happening in this case.